notebooks/en/pipeline_notus_instructions_preferences_legal.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>Few suggestions:</p><ul><li>Add a link to inference edpoints and a brief description. </li><li>Use non abbreviated version of "RAG" when first used  i.e. "Retreival Augmented Generation" before switching to abbreviation.  </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>